### DISCUSSION TITLE
aarch64 build

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#836] [Ideas]
### What feature would you like to see?

I would like to see an aarch64 build.

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/438_

**Comment by youwen5** [comment]
Very interested in being able to run this on my Asahi Linux macbook, any specific blockers preventing aarch64 support or is it just something that hasn't been heavily looked at yet?

**Reply by Heinrich-XIAO** [reply]
It would also be able to run on raspberry pis
